librz/bin/format/ne/ne.c

```diff
@@ -118,6 +118,44 @@ static void ne_sanitize_name(char *name, ut16 count) {
 	}
 }
 
+static const char *get_reloc_type_name(const ut8 src_type, const ut8 flag) {
+#define CONCAT(a, b) a b
+#define NE_RELOC_TARGET_TYPE(src_type_name, flag) \
+	switch (flag) { \
+	case INTERNAL_REF: \
+		return CONCAT(src_type_name, "_INTERNAL_REF"); \
+	case IMPORTED_ORD: \
+		return CONCAT(src_type_name, "_IMPORTED_ORD"); \
+	case IMPORTED_NAME: \
+		return CONCAT(src_type_name, "_IMPORTED_NAME"); \
+	case OSFIXUP: \
+		return CONCAT(src_type_name, "_OSFIXUP"); \
+	case ADDITIVE: \
+		return CONCAT(src_type_name, "_ADDITIVE"); \
+	default: \
+		return CONCAT(src_type_name, "_UNKNOWN"); \
```

Comment:
http://benoit.papillault.free.fr/c/disc2/exefmt.txt
seems we need to apply a mask?

Reply:
I think you're right, the documentation I've used must've been inaccurate